signingscript/src/signingscript/task.py

```diff
@@ -185,7 +193,7 @@ def _sort_formats(formats):
     """
     # Widevine formats must be after other formats other than macapp; GPG must
     # be last.
-    for fmt in ("widevine", "autograph_widevine", "autograph_omnija", "macapp", "autograph_rsa", "autograph_gpg"):
+    for fmt in ("widevine", "autograph_widevine", "stage_autograph_widevine", "autograph_omnija", "stage_autograph_omnija", "macapp", "autograph_rsa", "autograph_gpg", "stage_autograph_gpg"):
```

Comment:
Ah; I think I originally didn't add this because it wasn't in the hardcoded list of `stage` formats, but now that those are gone it definitely makes sense to update this...